admin/resources/js/components/PoolDashboard.tsx

```diff
@@ -195,6 +202,12 @@ const routes: Routes<RouterResult> = [
       component: <UpdatePool poolId={params.id as string} />,
     }),
   },
+  {
+    path: departmentTablePath(),
+    action: () => ({
+      component: <DepartmentPage />,
+    }),
+  },
```

Comment:
Missing objects in routes for the following:
- `departmentCreatePath`
- `departmentUpdatePath`